### ISSUE TITLE
for keybon companion app, be able to disable switch layouts when window focus changes

### ISSUE THREAD

**Issue by b4wii** [CoretechR/Keybon#12]
trying to use this as a custom streamdeck, to be used with obs. but as the current iteration of keybon, there is no way to change the default screen, except changing the sketch in the Arduino ide, and there is no way to have the companion not switch layouts when the focus of the current window changes. 

would be nice to have a toggle function to keep a certain layout active full-time, regardless of what window is active. 
I'd edit the companion app myself, but I do not know enough c# to be able to add this functionality. 

I realize that this is not a use for what keybon was originally intended for, but I think it could be used like a stream deck, just without all the fancy features of a stream deck. implemented. 
I just would like to be able to have it send a few certain button presses for hotkeys for OBS Studio, but with the current version of keybon this is pretty much impossible without doing some major coding on my part.

**Comment by CoretechR** [comment]
That's a good idea. I never really thought about that but it would make sense.
Have you already built a working Keybon?

**Comment by b4wii** [comment]
Yes. And when trying to set up companion it was kinda frustrating having it switch layouts all the time. 

Ooops didn't mean to close this.

**Comment by CoretechR** [comment]
[keybon companion b4wii.zip](https://github.com/CoretechR/Keybon/files/13782160/keybon.companion.b4wii.zip)
I made some quick changes to allow modifying the default page. Could you try and see if that works for you?

**Comment by b4wii** [comment]
okay, while it does let me make changes on the default screen, I don't seem to be able to make it function with the F13-F24 keys, even though sendkeys supports those keys. and that is what my hotkeys are set up for in my different programs.

**Comment by CoretechR** [comment]
That's strange. The function keys seem to work for me using {F5} for example. Can you try with this version again, please?
[keybon companion b4wii.zip](https://github.com/CoretechR/Keybon/files/13787915/keybon.companion.b4wii.zip)

**Comment by b4wii** [comment]
okay, in testing, it seems to stop at F16. anything beyond for the f17-f24 does not work. 

https://learn.microsoft.com/en-us/previous-versions/troubleshoot/winautomation/product-documentation/best-practices/valid-key-codes-for-the-send-keys-action

supposedly these are supported according to these tables.  unless I am reading wrongly.

**Comment by CoretechR** [comment]
Oh, I thought all the function keys don't work. I must have misread that. Depending on where you look, the documentation lists F16 as the last working function key with the sendkeys method:
https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.sendkeys.send?view=windowsdesktop-8.0

**Comment by b4wii** [comment]
That could be the case and makes sense. Kinda disappointed that send keys only seems to support up to f16.
